ui/src/main/java/io/jmix/ui/screen/StandardEditor.java

```diff
@@ -547,29 +547,32 @@ protected void commit(@SuppressWarnings("unused") Action.ActionPerformedEvent ev
         commitChanges()
                 .then(() -> {
                     commitActionPerformed = true;
-                    showSaveNotification();
+                    if (showSaveNotification) {
```

Comment:
What is the purpose of checking `showSaveNotification` outside of the method?

Reply:
Because if you want to override this method, you'll need to copy and paste this condition, otherwise the flag won't work and the editor's behavior becomes inconsistent.

Reply:
On the other hand, you can show notification regardless of the `showSaveNotification` flag if you don't aware that it is your job to check this flag. Since the `showSaveNotification()` method have a distinct purpose, I suppose that it's better to have a single place for such check, rather than copy-paste if-condition every time you need to show the notification.